include/sys/dtb.h

```diff
@@ -0,0 +1,11 @@
+#ifndef _SYS_DTB_
+#define _SYS_DTB_
+
+#include <sys/types.h>
+
+void *dtb_root(void);
```

Comment:
I've checked other files in `include/sys/` and please add some bigger comment about what it is about or at lease smaller to every line

Reply:
done